engine/codeflow_engine/actions/llm/manager.py

```
@@ -16,7 +16,11 @@
 from codeflow_engine.actions.llm.types import LLMResponse
 from codeflow_engine.core.llm.sluice import (SluiceMetadataError,
                                              SluiceNotConfiguredError)
-from codeflow_engine.core.llm.sluice_provider import SluiceProvider
+# Aliased: two Sluice providers exist. This one takes a typed SluiceAgent from
+# the closed set; the config-driven one above normalises a free-form `agent` off
+# the request. Importing both under the bare name silently shadowed one of them.
+from codeflow_engine.core.llm.sluice_provider import (SluiceProvider as
+                                                     TypedSluiceProvider)
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)